src/app/child-dev-project/notes/model/note.ts

```diff
@@ -52,7 +52,8 @@ export class Note extends Entity {
   @DatabaseField() date: Date;
   @DatabaseField() subject: string = "";
   @DatabaseField() text: string = "";
-  @DatabaseField() author: string = "";
+  /** IDs of authors of this note */
+  @DatabaseField() authors: string[] = [];
```

Comment:
maybe we could use a getter here to automatically migrate the old "author" string to the new format here?

Or we will need a one-time migration script to convert the properties. That could be done semi-manually - triggered from the admin section like [AttendanceMigrationService](https://github.com/Aam-Digital/ndb-core/blob/master/src/app/child-dev-project/attendance/attendance-migration/attendance-migration.service.ts).